modules/appeals_api/app/swagger/appeals_api/v1/api_description.md

```diff
@@ -43,6 +43,16 @@ The submission statuses begin with pending and end with success.
 | success   | The centralized mail portal, Digital Mail Handling System (DHMS), has received the data. |
 | error   | An error occurred. See the error code and message for further information. |
 
+#### Status Simulation
+
+Sandbox test submissions do not progress to Central Mail / VBMS.  In the lower environments (i.e. Sandbox or Staging), the final status of a submission is either `submitted` (for NOD) or `received` (for HLR). In the lower environments, we allow passing a `Status-Simulation` header on the show endpoints so that you can simulate the other statuses.
```

Comment:
Based on my E2E testing, this is factually incorrect. What actually happens is that it hits the environment, but CMP (and VBMS, I guess) does receive it, but doesn't take action with it. (I know this is the language used by VBA, but i'm wondering if we should clarify.)